conda/conda-build/meta.yaml

```diff
@@ -10,7 +10,7 @@
 ## The placeholder version is strictly for making two-pass conda build process.
 ## It should not be used for any other purpose, and this is not a default version.
 {% set placeholder_version = '0.0.0.dev' %}
-{% set default_cuda_version = '11.8' %}
+{% set default_cuda_version = '12.0' %}
```

Comment:
I think the legate.core default version also needs to be bumped to 12.0 in meta.yaml